drivers/virtualbox/virtualbox_test.go

```diff
@@ -29,3 +226,27 @@ func TestGetRandomIPinSubnet(t *testing.T) {
 		t.Fatalf("expected third octet of %d; received %d", testIP[2], newIP[2])
 	}
 }
+
+func newDriver(name string) *Driver {
+	return NewDriver(name, "")
+}
+
+// create parses the provided args and tries to configure the driver for
+// a `create` action.
+func create(args []string) (*Driver, error) {
```

Comment:
Then what's your idea on testing that flag definition and flag reading are aligned and working as expected. This can't be done all in integration tests